lib/chef/resource/execute.rb

```diff
@@ -101,6 +102,13 @@ def group(arg=nil)
         )
       end
 
+      def live_stream(arg=nil)
+        set_or_return(
+          :live_stream,
+          arg,
+          :kind_of => [ TrueClass, FalseClass ])
```

Comment:
Never mind, I see it does above. Any reason to not use the `default:` key with set_or_return?